tests/tests/test-fees/test-fee-multiplier.ts

```diff
@@ -314,24 +314,6 @@ describeDevMoonbeam("Fee Multiplier - XCM Executions", (context) => {
           },
         },
       })
-      .push_any({
-        Transact: {
-          originType: "SovereignAccount",
-          requireWeightAtMost: new BN(1000000000),
-          call: {
-            encoded: transferCallEncoded,
-          },
-        },
-      })
-      .push_any({
-        Transact: {
-          originType: "SovereignAccount",
-          requireWeightAtMost: new BN(1000000000),
-          call: {
-            encoded: transferCallEncoded,
-          },
-        },
-      })
```

Comment:
We are removing this multiple Transact instructions from this tests because they are not adapted yet to build and send V3 fragments - other option is to just skip them. V3 will require 64kb proofsize for each Transact, and we cannot buy execution for all of them (as the V2->V3 conversion will just add 64kb max weight).

Ideally as a followup we might want to adapt and run this tests using both XCM versions.

@timbrinded to confirm removing them does not affect the test itself

Reply:
Yup that's fine. 

We only had both txns there so that we could test that both v1 and v2 XCM executions wouldn't affect dynamic fee. Like you said, better to just write new tests in the future to send it as_V3